elasticapm/base.py

```diff
@@ -413,16 +418,21 @@ def _build_msg_for_logging(self, event_type, date=None, context=None, custom=Non
         log = event_data.get('log', {})
         if stack and 'stacktrace' not in log:
             if stack is True:
-                frames = stacks.iter_stack_frames()
+                frames = stacks.iter_stack_frames(skip=3)
```

Comment:
why do we need this now?

Reply:
It's a bug I noticed when looking at the frames in the test. It included some frames from the agent, so the first frame wasn't where you actually had the `logger` call.

I suspect there are more issues with too many frames in other places we collect them, but I'll leave that for another PR.